templates/tools/dockerfile/test/sanity/Dockerfile.template

```diff
@@ -14,28 +14,22 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
-  FROM debian:jessie
-  
-  <%include file="../../apt_get_basic.include"/>
-  <%include file="../../gcp_api_libraries.include"/>
-  <%include file="../../python_deps.include"/>
+  <%include file="../../python_stretch.include"/>
   <%include file="../../cxx_deps.include"/>
   #========================
   # Sanity test dependencies
+  RUN apt-get update && apt-get -t testing install -y python3.7 python3-all-dev
+  RUN curl https://bootstrap.pypa.io/get-pip.py | python3.7
   RUN apt-get update && apt-get install -y ${"\\"}
-        python-pip ${"\\"}
         autoconf ${"\\"}
         automake ${"\\"}
         libtool ${"\\"}
         curl ${"\\"}
-        python-virtualenv ${"\\"}
-        python-lxml ${"\\"}
         shellcheck
-  RUN pip install simplejson mako
-  
+  RUN python2.7 -m pip install simplejson mako virtualenv lxml
```

Comment:
Is there a way we can factor out this package list to a string so we don't have to repeat it?

Reply:
Many build scripts install their ad-hoc list of packages, I agree with you that we should have several `requirements.txt` placed in the project, and install packages accordingly.